app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -51,10 +51,12 @@ def label_tag_mapping_edit
   end
 
   def entity_ui_name_or_all(entity)
-    if entity.nil?
-      _("<All>")
-    else
+    if entity
+      entity = entity.split('::').last
+      entity = 'VmOrTemplate' if entity == 'Image'
```

Comment:
so 'Amazon::Vm' becomes `ui_lookup(model: 'Vm') == "VM and Instance"`,
`Amazon::Image` becomes `ui_lookup(model: 'VmOrTemplate') == "VM or Template"`.
Would user understand the distinction?  Perhaps you want something like `_("Vm")` and `_("Image")`?